packages/sdk/src/Confidence.ts

```diff
@@ -390,7 +402,7 @@ function defaultFetchImplementation(): typeof fetch {
 function defaultLogger(): Logger {
   try {
     if (process.env.NODE_ENV === 'development') {
-      return Logger.withLevel(console, 'info');
+      return Logger.withLevel(console, 'debug');
```

Comment:
Was this a conscious decision?

Reply:
yes! this way we default to debug logger in our demo apps and can see the logs when testing

Reply:
yeah that makes sense. I'll change it back to info